app/helpers/rad_common/application_helper.rb

```diff
@@ -149,6 +149,12 @@ def bootstrap_flash_close_button
       tag.button(sanitize('&times;'), type: 'button', class: 'close', 'data-dismiss': 'alert')
     end
 
+    def table_row_style(record, style_class: 'table-danger')
+      return unless record.present? && record.respond_to?(:active?) && !record.active?
```

Comment:
should we remove the `record.respond_to?(:active?)` and just require that we always implement `active?` on every model so we never forget any of these?